makedoc.g

```diff
@@ -48,39 +48,8 @@ for i in [ 1 .. Length(tst) ] do
 od;
 end;
 
-###########################################################################
```

Comment:
The function above is unused now - my plan is to clean it up when we will incorporate AutoDoc's `maketest.g` functionality into testing.